internal/dag/builder.go

```diff
@@ -39,17 +39,8 @@ const (
 type Builder struct {
 	KubernetesCache
 
-	// ExternalInsecurePort is the port that HTTP
-	// requests will arrive at the ELB or NAT that
-	// presents Envoy at the edge network.
-	// If not supplied, defaults to 80.
-	ExternalInsecurePort int
+	// TODO: Remove this layer of indirection?
```

Comment:
// TODO(youngnick) merge Builder and Kubernetes cache? See #nnnn

Reply:
The TODO doesn't mean you'll do it, it just means you're the person who knows best what that comment means.